src/components/checkboxes/_checkboxes.scss

```diff
@@ -14,7 +14,7 @@
     width: 100%;
 
     &:last-child {
-      margin-bottom: 0;
+      margin-bottom: 1rem;
```

Comment:
Adds too much space after answer options now (see below).  I think the panel padding itself should probably control the spacing.  Could we do something that targets the last element of any type within a panel to remove any bottom margin? We want 1rem spacing all around the error panel.  
![Screenshot 2020-09-10 at 11 36 09](https://user-images.githubusercontent.com/43346934/92719444-0a622880-f35b-11ea-94b7-793a7c5b9043.png)

Reply:
Ok yeah I can have a look at that. I only did it this way because I saw that all the other components had a 1rem bottom margin plus the panel padding so was just keeping it consistent. Ill see if I can do that and remove it for all in panels